queues.go

```diff
@@ -1,68 +1,109 @@
 package control
 
+// Region is an enum of the possible queue regions.
```

Comment:
I am wondering if this is a bit misdirecting, as Region is string in this case, not an enum.

Reply:
`type Region string` defines a new type called `Region` that is backed by a string. The goal of the type is so users see the region type as an enum of specific values and not an arbitrary string.

Reply:
Ah, makes sense.